eth/protocol.go

```diff
@@ -60,6 +61,10 @@ const (
 	NodeDataMsg        = 0x0e
 	GetReceiptsMsg     = 0x0f
 	ReceiptsMsg        = 0x10
+
+	// New protocol message codes introduced in eth65
+	NewTxHashesMsg = 0x11
+	GetTxsMsg      = 0x12
```

Comment:
I think we can reuse 0x08 and 0x09. No need to leave that gap there indefinitely.